CHANGES.rst

```diff
@@ -1,27 +1,27 @@
 Changes
 =======
 
-Version 0.8.0 (UNRELEASED)
+Version 0.8.0 (2021-11-22)
 --------------------------
 
-- Adds new command ``quota-show``.
-- Adds possibility to filter by workflow status and search by workflow name to ``list`` command.
-- Adds job start and finish times to ``logs`` command output.
-- Adds possibility to filter by file name, size and last modified values to ``ls`` command.
-- Adds new command ``info`` that retrieves general information about the cluster.
+- Adds new command ``quota-show`` to retrieve information about total CPU and Disk usage and quota limits.
+- Adds new command ``info`` that retrieves general information about the cluster, including the list all the available workspace paths.
 - Adds new ``--server-capabilities`` flag to ``validate`` command.
-- Adds the possibility to filter and display all the runs of a given workflow to the ``list`` command.
-- Changes ``delete`` to prevent workflow hard deletion.
-- Changes ``list`` to display interactive session status.
+- Adds support for ``outputs.directories`` in ``reana.yaml``.
+- Adds support for Snakemake workflow submission and validation.
```

Comment:
- Adds support for running and validating Snakemake workflows.

(I guess you can make it new number one in the list, as this is the biggest new feature!) Then the output directories, and then the rest of items will use the usual adds new comand, changes foo command stucture.)